rules/set-systctls-params/raw.rego

```diff
@@ -0,0 +1,74 @@
+package armo_builtins
+
+### POD ###
+
+# Fails if securityContext.systctls is not set
+deny[msga] {
+    # verify the object kind
+	pod := input[_]
+	pod.kind = "Pod"
+
+	# check securityContext has systctls set
+    not pod.spec.securityContext.systctls
+
+    path := "spec.securityContext.systctls"
+    msga := {
+		"alertMessage": sprintf("Pod: %v does not set 'securityContext.systctls'", [pod.metadata.name]),
+		"packagename": "armo_builtins",
+		"alertScore": 7,
+		"failedPaths": [path],
+		"fixPaths": [],
```

Comment:
we should return a fixPath here instead of failedPath. Is there a recommended value we can give the user here?

Reply:
> Is there a recommended value we can give the user here?

The CIS just say to defined as appropriate :/